core-bundle/src/Twig/Runtime/BackendHelperRuntime.php

```diff
@@ -21,4 +22,24 @@ public function icon(string $src, string $alt = '', HtmlAttributes|null $attribu
 
         return $image->getHtml($src, $alt, $attributes ? $attributes->toString(false) : '');
     }
+
+    /**
+     * Returns a file icon based on the mapping in the TL_MIME superglobal.
+     */
+    public function file_icon(FilesystemItem $item, string $alt = '', HtmlAttributes|null $attributes = null): string
```

Comment:
Shouldn't we try to deprecate this superglobal first rather than building new functionality on top of it?
There are so many things in Symfony building on top of the `MimeTypes` logic that I think we should just take this. As far as I could see it's even the idea that you could provide a default list (overriding the core list) in the constructor of `MimeTypes`, it's just not configurable in the framework yet: https://github.com/symfony/framework-bundle/blob/c241fc62c1d4b2b77a8686885a8e04dd6173feb9/Resources/config/mime_type.php#L18